File: atcoder/abs/1.swift
//usr/bin/env swift $0 $@;exit
let arr=readLine()!.characters.split{$0==" "}.map{Int(String($0))!}
let a=arr[0]
let b=arr[1]
print(a*b%2>0 ? "Odd" : "Even")

File: atcoder/abs/1.vb
module ABC086A
	sub Main(ByVal args() as String)
		dim a,b as integer
		dim arr(2) as string
		arr=Split(Console.ReadLine())
		a=arr(0):b=arr(1)
		Console.WriteLine(If(a*b mod 2 > 0,"Odd","Even"))
	end sub
end module
